scripts/Create_table_mh_intensity_canada_v2.sql

```diff
@@ -1,53 +1,46 @@
 
--- version 2 replaces old version with updated references, old column names are retained from v1 to keep scripts running correctly.
-
+-- version 3 replaces previous version, methodology and data changes.  Name kept the same to keep scripts from running.
 DROP TABLE IF EXISTS mh.mh_intensity_canada CASCADE;
 
 -- create table
 CREATE TABLE mh.mh_intensity_canada(
-    PRIMARY KEY(SAUIDt),
-    OBJECTID float,
-    SAUIDt varchar,
-    lon float,
-    lat float,
-    PGV float,
-    PGA float,
-    Sa0p1 float,
-    Sa0p3 float,
-    Sa1p0 float,
-    Sa2p0 float,
-    Vs30 float,
-    MMI6 float,
-    MMI7 float,
-    MMI8 float,
-    Tsun_ha float,
-    Fl200 float,
-    Fl500 float,
-    Fl1000 float,
-    Fire float,
-    Lndsus float,
-    Cy100 float,
-    Cy250 float,
-    Cy500 float,
-    Cy1000 float,
-    PGVn float,
-    MMI7n float,
-    Tsun_n float,
-    Fl500n float,
-    Firen float,
-    Lndn float,
-    Cy500n float,
-    MHn_TPGV float,
-    MHn_TMMI7 float,
-    MHIn_PGV float,
-    MHIn_MMI7 float
+PRIMARY KEY(sauidt),
+sauidt varchar,
+sauidi integer,
+lon float,
+lat float,
+pgv2500 float,
+pgv500 float,
+pga2500 float,
+pga500 float,
+vs30 float,
+mmi6 float,
+mmi7 float,
+mmi8 float,
+tsun500 float,
+fld50_jrc float,
+fld100_jrc float,
+fld200_jrc float,
+fld500_jrc float,
+fld200_unep float,
+fld500_unep float,
+fld1000_unep float,
+wildfire float,
+wui_type varchar,
+lndsus float,
+cy100 float,
+cy250 float,
+cy500 float,
+cy1000 float,
+svlt_score integer
+
 );
 
 /*
 -- import exposure from csv
-COPY mh.mh_intensity_canada (OBJECTID, SAUIDt, lon, lat, PGV, PGA, Sa0p1, Sa0p3, Sa1p0, Sa2p0, Vs30, MMI6, MMI7, MMI8, Tsun_ha, Fl200, Fl500, Fl1000, Fire, Lndsus, Cy100, Cy250, Cy500, Cy1000, PGVn, MMI7n, Tsun_n, Fl500n, Firen, Lndn, Cy500n, MHn_TPGV, MHn_TMMI7, MHIn_PGV, MHIn_MMI7)
-
-    FROM '/usr/src/app/mh-intensity-sauid.csv'
+COPY mh.mh_intensity_canada(sauidt,sauidi,lon,lat,pgv2500,pgv500,pga2500,pga500,vs30,mmi6,mmi7,mmi8,tsun500,fld50_jrc,fld100_jrc,fld200_jrc,fld500_jrc,fld200_unep,fld500_unep,fld1000_unep,wildfire,wui_type,lndsus,cy100,cy250,
+cy500,cy1000,svlt_score)
+    FROM 'D:\Workspace\data\source datasets\mh\HTi_sauid_2021.csv'
```

Comment:
Might need to change `D:\Workspace\data\source datasets\mh\` to `/usr/src/app/` or something like that?

@wkhchow, I wonder if the COPY ... FROM statement can interpret a relative path reliably such that you could use the exact same path on your Windows machine as well as in the final Docker image for add_data.sh run?

Reply:
The code is commented out but we can remove it to avoid confusion.